javascript/node/selenium-webdriver/chrome.js

```diff
@@ -284,7 +284,6 @@ function getDefaultService() {
 Options.prototype.CAPABILITY_KEY = 'goog:chromeOptions'
 Options.prototype.BROWSER_NAME_VALUE = Browser.CHROME
 Driver.getDefaultService = getDefaultService
-Driver.prototype.VENDOR_COMMAND_PREFIX = 'goog'
```

Comment:
The vendor prefix is still being used on Chromium based browsers like Edge Chromium and Chrome. Did you mean to remove this?

Reply:
I have overlooked into it. I have reverted the change. Thank you for review @AutomatedTester !